### DISCUSSION TITLE
can u help me?

### DISCUSSION THREAD

**Discussion by Mejo33** [shekohex/coemu#54] [General]
where can i check user and pass for sqlite database? how can i add new account entry?

**Comment by shekohex** [comment]
Sqlite does not have a user and password by default so you can open it in supported Database Managers.

To Create an Account, you should open the Database on a Database Manager (something like [Datagrip](https://www.jetbrains.com/datagrip/) or [alternatives](https://www.slant.co/options/210/alternatives/~datagrip-alternatives)) then in the `accounts` table you should create a new raw with your account information, you should only need to input the `username` and `password`.
Please note that the `password` needs to be hashed using [Bcrypt](https://en.wikipedia.org/wiki/Bcrypt) .. You could use the provided tool to get a hashed password, just run

```bash
$ cargo hash-pwd <password>
```

for example

```
$ cargo hash-pwd test
$2b$12$iSrnkacd/i/8eZr5pBoDlO5qcbLmLUWGQ6IN.oQuemnlRKU/NExIW
```

After creating the account, you should be able to login and create your character :)

**Comment by Mejo33** [comment]
https://i.imgur.com/T0tyNSK.png

start server login and game and when i trying login in empty account its show me character creation window but when i create charater and try to login ingame i see this erro in console. Any help?

**Reply by shekohex** [reply]
This means the newbie map is not found. Have you followed the steps mentioned here?
 https://github.com/shekohex/coemu/tree/master/data/GameMaps#readme

**Reply by Mejo33** [reply]
https://i.imgur.com/FyzfEl6.png

It works! thanks for your help.

**Comment by cfsalas-a** [comment]
Español please

**Comment by cfsalas-a** [comment]
Discord?

**Reply by shekohex** [reply]
There is no discord, sorry. How can I help you?
